apps/cowswap-frontend/src/legacy/state/swap/hooks.tsx

```diff
@@ -225,19 +224,13 @@ export function useUnknownImpactWarning(priceImpactParams?: PriceImpact) {
 
 function _computeUnknownPriceImpactAcceptedState({
   impactWarningAccepted,
-  priceImpactParams,
   isExpertMode,
 }: {
   impactWarningAccepted: boolean
   priceImpactParams?: PriceImpact
   isExpertMode: boolean
 }) {
   if (isExpertMode || impactWarningAccepted) return true
-  else {
-    if (priceImpactParams?.error) {
-      return impactWarningAccepted
-    }
-  }
```

Comment:
Oh, don't we need this warning?
Is it being captured somewhere else?

Reply:
You saved me, thanks!
Fixed it in the next PR:
https://github.com/cowprotocol/cowswap/pull/3081/commits/d0019d6f6c7ed3f09f77ab4126abf6d4e0c46011